ledger/complete/ledger.go

```diff
@@ -77,11 +77,15 @@ func NewLedger(dbDir string,
 		pathFinderVersion: pathFinderVer,
 	}
 
+	// pause records to prevent double logging trie removals
+	w.PauseRecord()
 	err = w.ReplayOnForest(forest)
 	if err != nil {
 		return nil, fmt.Errorf("cannot restore LedgerWAL: %w", err)
 	}
 
+	w.UnpauseRecord()
```

Comment:
could use defer pattern, so that we unpause even if theres err

Reply:
done at bb3bcac.